js/plugin/Routing.js

```diff
@@ -252,7 +252,9 @@ BR.Routing = L.Routing.extend({
     // change segment color before request to indicate recalculation (mark old)
     if (m1 && m1._routing.nextLine !== null) {
         m1._routing.nextLine.options.color = 'dimgray';
-        m1._routing.nextLine._updateStyle();
+
+        console.log("TODO: fixme! //m1._routing.nextLine._updateStyle()");
+        // m1._routing.nextLine._updateStyle();
```

Comment:
Another thing I wasn't able to fix. Due to leaflet upgrade from 0.7.x to 1.x, this function appears not to exist anymore but… any help welcome about how to fix that!